lib/migration.h

```diff
@@ -46,11 +46,22 @@
 
 struct migration *
 init_migration(const vfu_migration_callbacks_t *callbacks,
-               uint64_t data_offset, int *err);
+               uint64_t flags, int *err);
+
+bool
```

Comment:
same comment I had re dma.h

Reply:
Okay, but what about `handle_mig_data_read/write`? Are they okay to stay in `migration.c` or should they be shifted out to `libvfio-user.c` too?

Reply:
that makes sense in migration.c IMO, since it's dealing with migration-specific stuff. It's a blurred line I will concede.